<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logbook::Internal::LogbookFormDialog</class>
 <widget class="QWidget" name="Logbook::Internal::LogbookFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>305</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelDatetime">
      <property name="text">
       <string>Date:</string>
      </property>
      <property name="buddy">
       <cstring>dateTimeEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelCallsign">
      <property name="text">
       <string>Callsign:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditCallsign</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditCallsign"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditName</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditName"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelMode">
      <property name="text">
       <string>Mode:</string>
      </property>
      <property name="buddy">
       <cstring>comboBoxMode</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelFrequency">
      <property name="text">
       <string>Frequency:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditFrequency</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditFrequency"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelBand">
      <property name="text">
       <string>Band:</string>
      </property>
      <property name="buddy">
       <cstring>comboBoxBand</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelRstSend">
      <property name="text">
       <string>RST send:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditRstSend</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditRstSend"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelRstRcvd">
      <property name="text">
       <string>RST rcvd:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditRstRcvd</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEditRstRcvd"/>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="pushButtonSubmitLogbookForm">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBoxBand"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBoxMode"/>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="displayFormat">
       <string>dd.MM.yy HH:mm:ss</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>lineEditCallsign</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxMode</tabstop>
  <tabstop>lineEditFrequency</tabstop>
  <tabstop>comboBoxBand</tabstop>
  <tabstop>lineEditRstSend</tabstop>
  <tabstop>lineEditRstRcvd</tabstop>
  <tabstop>pushButtonSubmitLogbookForm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>test()</slot>
 </slots>
</ui>
